core-common/src/main/java/org/zalando/nakadi/repository/kafka/EventAuthHeader.java

```diff
@@ -0,0 +1,27 @@
+package org.zalando.nakadi.repository.kafka;
+
+
+import com.google.common.base.Charsets;
+import org.apache.kafka.common.header.Header;
+import org.apache.kafka.common.header.internals.RecordHeader;
+
+public class EventAuthHeader extends RecordHeader {
+
+    static final String HEADER_KEY = "X-Event-Auth";
+
+    public EventAuthHeader(final String value) {
+        super(HEADER_KEY, value.getBytes(Charsets.UTF_8));
```

Comment:
If the `value` is `null`, there will be `NPE` here, but I guess, the value should be check before instantiating the header (before publishing), right?

Reply:
I re read the api docs, it looks like it could be null
anyway, thanks 
ebf5b05